posthog/celery.py

```diff
@@ -428,15 +429,31 @@ def check_cached_items():
     update_cached_items()
 
 
-@app.task(ignore_result=False)
-def update_cache_item_task(key: str, cache_type, payload: dict) -> None:
+@app.task(ignore_result=True)
+def update_cache_consumer_rate_limit(rate_limit: Optional[Union[str, int]]) -> None:
+    from structlog import get_logger
+
+    logger = get_logger(__name__)
+
+    if rate_limit is not None:
+        logger.info("set_update_cache_consumer_rate_limit", rate_limit=rate_limit)
+        app.control.rate_limit("posthog.celery.update_cache_item_task", rate_limit)
```

Comment:
This takes care of updates, what about on initialisation? If I'm reading these changes right, on load we will use the default set on line 443 of "6/m" and then only if the instance setting is updated will we change it to something else. Feels like we need to read the instance setting on startup?

Reply:
Ah! Much better 💖